src/reducers/entities/posts.js

```diff
@@ -31,6 +31,30 @@ function handleReceivedPost(posts = {}, postsInChannel = {}, action) {
     return {posts: nextPosts, postsInChannel: nextPostsForChannel};
 }
 
+function handleRemovePendingPost(posts = {}, postsInChannel = {}, action) {
```

Comment:
Might be good to have test for `handleRemovePendingPost`.

Reply:
You are more than right again :)

Reply:
Done! :)